hsmd/hsmd.c

```diff
@@ -434,7 +434,7 @@ static struct io_plan *init_hsm(struct io_conn *conn,
 	struct secret *hsm_encryption_key;
 	struct bip32_key_version bip32_key_version;
 	u32 minversion, maxversion;
-	const u32 our_minversion = 4, our_maxversion = 5;
+	const u32 our_minversion = 4, our_maxversion = 6;
```

Comment:
I am still thinking that this value should be equal to the constant values defined inside the `common/hsm_version.h`, but this is an off the book comment